shared/chat/conversation/messages/react-button/index.js

```diff
@@ -207,7 +207,7 @@ export class NewReactionButton extends React.Component<NewReactionButtonProps, N
                 style={iconCastPlatformStyles(
```

Comment:
probably related to higher level styles and not these changes, but the emoji picker jumps around when trying to choose an emoji on desktop. 
![picker-pogo](https://user-images.githubusercontent.com/11968340/49451881-1dbb8980-f7ae-11e8-8689-e1329d834457.gif)
(only clicked on the icon once in this, hovering on it again brings the picker back)